Description: Allow to send e-mail to multiple recipients.
Bug-Debian: http://bugs.debian.org/348756
Author: Daniel J. Priem <danielpriem@flexserv.de>
Last-Update: 2010-07-11

--- a/scripts/cs_helpers.pyin
+++ b/scripts/cs_helpers.pyin
@@ -243,7 +243,7 @@
 
         try:
                 server = smtplib.SMTP('localhost')
-                server.sendmail(mail_from,mail_to,msg.as_string())
+                server.sendmail(mail_from, mail_to.replace('"', '').split(',') ,msg.as_string())
                 server.quit()
         except Exception,e:
                 capisuite.error("Error while trying to send mail: %s" % e)
