--- a/acinclude.m4
+++ b/acinclude.m4
@@ -82,7 +82,11 @@ python_version=`${PYTHON} -c "import sys
 python_prefix=`${PYTHON} -c "import sys; print sys.prefix"`
 python_execprefix=`${PYTHON} -c "import sys; print sys.exec_prefix"`
 python_libdir=`${PYTHON} -c "from distutils import sysconfig; print sysconfig.get_python_lib(1,1)"`
-python_configdir="${python_libdir}/config"
+if test -d "${python_libdir}/config"; then
+	python_configdir="${python_libdir}/config"
+elif test -d "${python_libdir}/config-`dpkg-architecture -qDEB_BUILD_MULTIARCH`"; then
+	python_configdir="${python_libdir}/config-`dpkg-architecture -qDEB_BUILD_MULTIARCH`"
+fi
 python_moduledir="${python_libdir}/site-packages"
 python_moduleexecdir="${python_libdir}/site-packages"
 python_includespec="-I${python_prefix}/include/python${python_version}"
